src/js/_enqueues/admin/post.js

```diff
@@ -511,7 +511,7 @@ jQuery( function($) {
 			// See https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event.
 			return __( 'The changes you made will be lost if you navigate away from this page.' );
 		}
-	}).on( 'unload.edit-post', function( event ) {
+	}).on( 'pagehide.edit-post', function( event ) {
```

Comment:
I don't see a single reference to the jQuery-namespaced `unload.edit-post` event name in WPdirectory, so I don't see any back-compat concerns here.

Reply:
Note that post lock is restored upon `pageshow` via heartbeat. See https://github.com/WordPress/wordpress-develop/pull/5056#discussion_r1343245997